xline/src/main.rs

```diff
@@ -367,17 +368,17 @@ async fn main() -> Result<()> {
     debug!("name = {:?}", cluster_config.name());
     debug!("server_addr = {:?}", self_addr);
     debug!("cluster_peers = {:?}", cluster_config.members());
-    let mem_storage = Memory::new();
+    let mem_engine = MemoryEngine::new();
```

Comment:
Shall we make the engine generic instead of hard coding?

Reply:
I will fixed it in the next pr